service/cloudconfigv3/cloud_config_test.go

```diff
@@ -0,0 +1,191 @@
+package cloudconfigv3
```

Comment:
This is the new v3 service. I've highlighted the differences. Otherwise its copy paste of the v2 service.